TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/kafka/KafkaTestUtils.java

```diff
@@ -51,7 +51,6 @@ static Producer<String, byte[]> buildKafkaProducer(String bootstrapServers) {
             return new KafkaProducer(kafkaProps);
         } catch (Exception e) {
             log.atError().setCause(e).log();
-            System.exit(1);
```

Comment:
sorry.  looks like I put that in, probably from transitioning from a main() class into a developing unit test. Thanks.